docs/source/plugins/06_solver_hooks.rst

```diff
@@ -240,7 +240,14 @@ each Liquid-Liquid system plugin hooks. In the sequence, they are presented
 
 .. warning::
     The hooks described in this section must be implemented all of them or none of them. If just part of four hooks
-    are implemented, the |Alfasim| will show an error of configuration.
+    are implemented, the |alfasim| will show an error of configuration.
+
+.. note ::
+    Note that the all of the hooks related to the liquid-liquid system of Mechanistic Model can use same emulsion
+    viscosity model used internally by |alfasim|. It is made by helper function :cpp:func:`get_relative_emulsion_viscosity`.
```

Comment:
"It is made by helper function :cpp:func:`get_relative_emulsion_viscosity`."

I didn't fully understood this sentence. Could you rephrase it? Is it supposed to be something like "The emulsion viscosity model can be obtained by using the helper function :cpp:func:`get_relative_emulsion_viscosity`."?

Reply:
Yes, exactly. this is the idea.
I will rephrase it. It is a little confusing.